airbyte-integrations/connectors/destination-oracle/src/main/java/io/airbyte/integrations/destination/oracle/OracleDestination.java

```diff
@@ -44,6 +44,8 @@ public class OracleDestination extends AbstractJdbcDestination implements Destin
   public static final String ENCRYPTION_KEY = "encryption";
   public static final String ENCRYPTION_METHOD_KEY = "encryption_method";
 
+  public static final String JDBC_URL_PARAMS_KEY = "jdbc_url_params";
```

Comment:
Do we need to have this constant as public? if yes, please describe why

Reply:
The Constant is declared as public and final b/c we are accessing this in other classes as well like test classes.Similar declaration is done in other connectors like MySQL.